operators/jaeger/1.48.1/manifests/jaeger-operator.clusterserviceversion.yaml

```
@@ -0,0 +1,589 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "jaegertracing.io/v1",
+          "kind": "Jaeger",
+          "metadata": {
+            "name": "my-jaeger"
+          },
+          "spec": {
+            "strategy": "allInOne"
+          }
+        }
+      ]
+    capabilities: Deep Insights
+    categories: Logging & Tracing
+    certified: "false"
+    containerImage: quay.io/jaegertracing/jaeger-operator
```

Seems the imagetag is missing: https://github.com/k8s-operatorhub/community-operators/actions/runs/6072765170/job/16473755012?pr=3151#step:6:2462

```suggestion
    containerImage: quay.io/jaegertracing/jaeger-operator:1.48.1
```

---

done